pages/tech-board.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: JHipster Tech board
+permalink: /tech-board/
+sitemap:
+    priority: 0.7
+    lastmod: 2021-03-31T00:00:00-00:00
+---
+
+# JHipster Technical board
+
+JHipster is supported by a technical board that ensures the platform continues to reflect and serve its diverse and growing user community. 
+The Board discusses and makes recommendations regarding the roadmap and technical directions.
+
+The meetings review and advise on industry trends, market strategy, project strategy and roadmap, ecosystem development.
+
+This board helps the project by giving advice and feedback on how the project and the related technologies are going.
+For the same reason, the board helps target incoming technologies and help to build the future of the project.
+
+In the end, this should help the project to raise resources to make those choices real by focusing on specific features, helps on finding new contributors, and advertise on the next project decisions…
+
+The decisions and materials produced by the committee are public and used to benefit the community rather than private interests.
+
+The board meets quarterly, with the ability to meet more frequently if needed.
+
+## Board Membership
+
+Current members:
+* Julien Dubois (Microsoft)
+* Deepu K Sasidharan (Adyen)
+* Pascal Grimaud (Ippon technologies)
+* Matt Raible (Okta)
+* Paul Hinz (Entando)
+
+
+The members are chosen by the project leaders and should come from:
+* representative industry companies and leaders
+* representative organisations that use JHipster
+* open-source project leaders
+* Java champions
```

Comment:
Just a minor comment; I suggest maybe removing `Java Champions` from the list just to keep it open for everyone? Feel like maybe it will give a wrong sense that we prefer java based experts for this. Or do we? 🤔

Reply:
Yes I agree. JHipster is more than Java

Reply:
We can instead use community leaders or technology leaders

Reply:
Yes, the idea was more about targeting the leaders. I'm changing for a more neutral wording.